### ISSUE TITLE
python test/runtest fails on linux "undefined symbol: mpc_cmp_abs"

### ISSUE THREAD

**Issue by vinklein** [aleaxit/gmpy#238]
`python test/runtests.py` and `python test_cython/runtests.py` fails on linux see https://travis-ci.org/vinklein/gmpy/builds/518178149.

**Comment by vinklein** [comment]
To be more concise :
```
>>> import gmpy2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/vklein/.local/lib/python2.7/site-packages/gmpy2/__init__.py", line 1, in <module>
    from .gmpy2 import *
ImportError: /home/vklein/.local/lib/python2.7/site-packages/gmpy2/gmpy2.so: undefined symbol: mpc_cmp_abs
```

**Comment by vinklein** [comment]
The travis tests uses the libmpc-dev package which is an mpc 1.0.3-1 in xenial.
We have at least a consistency problem because `mpc_cmp_abs` is new in mpc-1.1.0 but gmpy2's INSTALL file says :
```
4. Download and un-tar the MPC source code. Change to MPC source directory
   and compile MPC.

    $ cd $HOME/src/mpc-1.0.3
    $ ./configure --prefix=$HOME/static --enable-static --disable-shared --with-pic --with-gmp=$HOME/static --with-mpfr=$HOME/static
    $ make
    $ make check
    $ make install
```

**Comment by casevh** [comment]
That would be perfect. For MPC 1.03, we should just call mpc_abs() on each of the complex arguments, and then do an mpfr_cmp_abs() on the results. I already have issue #236 open for this.

**Comment by casevh** [comment]
Fixed by PR #239.
